lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2809,33 +2809,28 @@ instance Arbitrary PartialTx where
             return (fromCardanoTxIn $ fst i, o, Nothing)
         let redeemers = []
         return $ PartialTx
-            (sealedTxFromCardano $ InAnyCardanoEra era tx)
+            tx
             resolvedInputs
             redeemers
     shrink (PartialTx tx inputs redeemers) =
         [ PartialTx tx inputs' redeemers
         | inputs' <- shrinkInputs inputs
         ] ++
         [ restrictResolution $ PartialTx
-            (sealedTxFromCardano $ InAnyCardanoEra Cardano.AlonzoEra tx')
+            tx'
             inputs
             redeemers
-        | tx' <- shrinkTx (alonzoCardanoTx tx)
+        | tx' <- shrinkTx tx -- TODO: Need to take tail to fix shrinker?
```

Comment:
👀

Reply:
Seems to be correct because we never use more than one `<-` in each list comprehension.